packages/lib/src/components/styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,85 @@
+import { Global, css } from "@emotion/react";
+import { fontFamilies } from "@leafygreen-ui/tokens";
+import { fontStyles } from "./fonts";
+
+export const resetStyles = css`
+  /* Reset styles, usage recommended by LeafyGreen. */
+  *,
+  *:before,
+  *:after {
+    box-sizing: border-box;
+  }
+`;
+
+const spruceResetStyles = css`
```

Comment:
My preference would probably be to not move all of these Spruce and Parsley specific styles to lib (unless there's a reason to do so that I'm missing). Instead, they can each import the base `resetStyles` and be used within each application.